portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -329,4 +348,19 @@ definition {
 		return "${getObjectsWithNestedFieldJSON}";
 	}
 
+	macro updateEmployee {
+		ObjectDefinitionAPI._updateEmployee(
+			employeeId = "${employeeId}",
+			firstname = "${employeeFirstname}",
+			newManagerId = "${newManagerId}");
+
+		var firstname = ObjectDefinitionAPI._getEmployeeFirstnameById(employeeId = "${employeeId}");
+
```

Comment:
Maybe it is not necessary to check the firstname here? In the end what we're trying to do is to check whether the updatedManager data appears.

Reply:
Remove the firstname check here and replace it with the updated manager data, or use code 204 instead

Reply:
@magjed4289 Delete the check for the name here, and then see if the return code 204, the update of the data can be reflected in the following method ObjectDefinitionAPI.assertEmployeeHasNestedFieldManager
I don't know if I understand correctly

Reply:
This is fine, no need to delete anything, sorry for the confusion! Idk why I thought that this was still in the update Method 🤦🏻‍♀️